CONTRIBUTING.md

```
@@ -226,34 +226,50 @@ To maintain a clean and linear project history, we prefer rebasing your feature
     git fetch origin develop
     ```
 
+
+    > **💡 GUI Alternative**: In VSCode, use the Source Control panel and click the refresh icon, or use the sync button in the status bar.
+
```

markdownlint-cli2 (0.17.2)

Blank line inside blockquote

(MD028, no-blanks-blockquote)